<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication3Class</class>
 <widget class="QMainWindow" name="QtGuiApplication3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>290</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1401</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionOpen_Image_2"/>
    <addaction name="actionOpen_Image_3"/>
    <addaction name="actionOpen_Image_4"/>
    <addaction name="actionOpen_Image_5"/>
    <addaction name="actionOpen_Image_6"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Cylindrical Projection</string>
    </property>
    <addaction name="actionCylindrical_Projection_image_1"/>
    <addaction name="actionInterpolation"/>
   </widget>
   <addaction name="menuopen"/>
   <addaction name="menuHistogram"/>
  </widget>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open Image 1</string>
   </property>
  </action>
  <action name="action45">
   <property name="text">
    <string>45</string>
   </property>
  </action>
  <action name="action90">
   <property name="text">
    <string>90</string>
   </property>
  </action>
  <action name="action135">
   <property name="text">
    <string>180</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_1">
   <property name="text">
    <string>Cylindrical Projection</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_2">
   <property name="text">
    <string>Cylindrical Projection_image 2</string>
   </property>
  </action>
  <action name="actionOpen_Image_2">
   <property name="text">
    <string>Open Image 2</string>
   </property>
  </action>
  <action name="actionOpen_Image_3">
   <property name="text">
    <string>Open Image 3</string>
   </property>
  </action>
  <action name="actionOpen_Image_4">
   <property name="text">
    <string>Open Image 4</string>
   </property>
  </action>
  <action name="actionOpen_Image_5">
   <property name="text">
    <string>Open Image 5</string>
   </property>
  </action>
  <action name="actionOpen_Image_6">
   <property name="text">
    <string>Open Image 6</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_3">
   <property name="text">
    <string>Cylindrical Projection_image 3</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_4">
   <property name="text">
    <string>Cylindrical Projection_image 4</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_5">
   <property name="text">
    <string>Cylindrical Projection_image 5</string>
   </property>
  </action>
  <action name="actionCylindrical_Projection_image_6">
   <property name="text">
    <string>Cylindrical Projection_image 6</string>
   </property>
  </action>
  <action name="actionInterpolation">
   <property name="text">
    <string>Interpolation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
